### PR TITLE
Run pytest as a GitHub action

### PR DESCRIPTION
**Pull request by patrys**
# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Currently can't fully replace Travis as the Django query tool does not support GitHub Actions.

**Comment by maarcingebala** [comment]
OK, so I think we want to keep Travis running Pytest until we figure out how to use performance benchmarks with actions.

**Comment by maarcingebala** [comment]
@patrys Should we close this one for now?

**Comment by patrys** [comment]
Tests fail because I've split #5847 into a separate PR

File: .github/workflows/pytest.yml
Comment (NyanKiyoshi) [inline .github/workflows/pytest.yml:31]:
```suggestion
      run: apt-get install -y libpq-dev
```

**Comment by maarcingebala** [comment]
https://github.community/t/run-actions-on-pull-requests-with-merge-conflicts/17104/14 - It seems that GH actions won't run if there are conflicts in the PR. We just stumbled upon this issue on the multichannel branch where we wanted to deploy the test environment (which is handled by a GH action) and now we're blocked until we resolve conflicts. Conflicts do happen especially on large feature branches that can stay behind master for a few days; I'm wondering if we still want to use them for things like tests that you would like to have running always.

**Comment by patrys** [comment]
@maarcingebala That's an interesting limitation I did not think about but I understand where it comes from: GitHub actions run on the result of a merge rather than on the tip of the branch which is something I wanted to have for a long time (as it means we're actually testing the resulting target branch, including all of the other merges that happened between the branching point and now). That's the only way to make sure tests will continue passing after the merge.

**Comment by NyanKiyoshi** [comment]
But seems like the rebase gone wrong for the branch you are talking about, there is 300 commits, it really needs a cleanup, I'm not sure what is happening in that PR.

**Review by NyanKiyoshi — APPROVED** [review]

**Review by maarcingebala — APPROVED** [review]

**Review by fowczarek — APPROVED** [review]

**Review by tomaszszymanski129 — APPROVED** [review]

**Pull request merged**
